concourse/steps/release.py

```diff
@@ -361,12 +367,16 @@ def _calculate_next_cycle_dev_version(
         version_operation: str,
         prerelease_suffix: str,
     ):
-        # calculate the next version and append the prerelease suffix
+        # calculate the next version
+        version_str = version.process_version(
+            version_str=release_version,
+            operation=version_operation,
+        )
+        if version_operation == version.NOOP:
```

Comment:
not sure I follow. but if the version-operation is a noop, thy not return it immediately (w/o first processing it)?